static/app/views/settings/organizationProjects/index.tsx

```diff
@@ -27,113 +31,106 @@ import ProjectStatsGraph from './projectStatsGraph';
-            stat: 'generated',
-            group: 'project',
-            per_page: ITEMS_PER_PAGE,
-          },
+      },
+    ],
+    {staleTime: Infinity}
+  );
+
+  const {data: projectStats} = useApiQuery<ProjectStats>(
+    [
+      `/organizations/${organization.slug}/stats/`,
+      {
+        query: {
+          since: time.current / 1000 - 3600 * 24,
+          stat: 'generated',
+          group: 'project',
+          per_page: ITEMS_PER_PAGE,
         },
-      ],
-    ];
-  }
-
-  getTitle(): string {
-    const {organization} = this.props;
-    return routeTitleGen(t('Projects'), organization.slug, false);
-  }
-
-  renderLoading(): React.ReactNode {
-    return this.renderBody();
-  }
-
-  renderBody(): React.ReactNode {
-    const {projectList, projectListPageLinks, projectStats} = this.state;
-    const {organization} = this.props;
-
-    const action = <CreateProjectButton />;
-
-    return (
-      <Fragment>
-        <SettingsPageHeader title="Projects" action={action} />
-        <SearchWrapper>
-          {this.renderSearchInput({
-            updateRoute: true,
-            placeholder: t('Search Projects'),
-            className: 'search',
-          })}
-        </SearchWrapper>
-        <Panel>
-          <PanelHeader>{t('Projects')}</PanelHeader>
-          <PanelBody>
-            {projectList ? (
-              sortProjects(projectList).map(project => (
-                <GridPanelItem key={project.id}>
-                  <ProjectListItemWrapper>
-                    <ProjectListItem project={project} organization={organization} />
-                  </ProjectListItemWrapper>
-                  <ProjectStatsGraphWrapper>
-                    {projectStats ? (
-                      <ProjectStatsGraph
-                        key={project.id}
-                        project={project}
-                        stats={projectStats[project.id]}
-                      />
-                    ) : (
-                      <Placeholder height="25px" />
-                    )}
-                  </ProjectStatsGraphWrapper>
-                </GridPanelItem>
-              ))
-            ) : (
-              <LoadingIndicator />
-            )}
-            {projectList && projectList.length === 0 && (
-              <EmptyMessage>{t('No projects found.')}</EmptyMessage>
-            )}
-          </PanelBody>
-        </Panel>
-        {projectListPageLinks && (
-          <Pagination pageLinks={projectListPageLinks} {...this.props} />
-        )}
-      </Fragment>
-    );
-  }
+      },
+    ],
+    {staleTime: Infinity}
+  );
+
+  const projectListPageLinks = getResponseHeader?.('Link');
+
+  const action = <CreateProjectButton />;
+
+  const debouncedSearch = debounce(
+    (searchQuery: string) =>
+      browserHistory.replace({
+        pathname: location.pathname,
+        query: {...location.query, query: searchQuery},
+      }),
+    DEFAULT_DEBOUNCE_DURATION
+  );
```

Comment:
wrap in a useCallback otherwise every rerender will create a new debounced function